packages/airnode-examples/README.md

```diff
@@ -2,17 +2,23 @@
 
 > A public list of examples showcasing the features of Airnode
 
-The project will contain many runnable scripts to guide you through the necessary steps. You are recommended to read the
-contents of the scripts as you run them.
+This project contains runnable scripts that will guide you through the necessary steps to run an Airnode instance. We
```

Comment:
I remember @wkande asking me to remove "we" and instead use the passive - but I am not sure why.